sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -10,23 +10,26 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2021 ScyllaDB
+import logging
 from difflib import unified_diff
-from typing import List, Literal, Union, Any
+from typing import Literal, List, Dict, Any, Union
 
-import logging
 import yaml
-from pydantic import validator, BaseModel, ConfigDict
-
-from sdcm.provision.scylla_yaml.auxiliaries import RequestSchedulerOptions, EndPointSnitchType, SeedProvider, \
-    ServerEncryptionOptions, ClientEncryptionOptions
+from pydantic import field_validator, BaseModel, ConfigDict
 
+from sdcm.provision.scylla_yaml.auxiliaries import EndPointSnitchType, ServerEncryptionOptions, ClientEncryptionOptions, \
+    SeedProvider, RequestSchedulerOptions
 
 logger = logging.getLogger(__name__)
 
 
 class ScyllaYaml(BaseModel):
+    """
+    Model for scylla.yaml configuration.
+    Allows for any parameters, parameters here are just type hint helpers
+    """
 
-    model_config = ConfigDict(arbitrary_types_allowed=True)
+    model_config = ConfigDict(arbitrary_types_allowed=True, extra="allow")
```

Comment:
This extra can be condition by some new simple SCT config option.
It could be "disallow" by default.
And when one needs to test some new Scylla config option not covered by SCT yet, he would be able to enable extras and use his target Scylla config option.

Reply:
It's not worth our time

if you want exact validation of the options, SCT can't be the source of truth.
It can be use as a helper for writing code in SCT, not more then that, that was the intention.

you can if you wish as a followup generate warning when there a miss-match. (which no on would be looking at)

SCT should pass-throw what user is asking, and in some cases what scylla is passing as default, i.e. new values.